recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
+
+package:
+  name: serpent
```

Comment:
The default channel use all lower case. I am slightly inclined to respect the original name as-is. But IMO that is no big deal as both conda and PyPI are case insensitive.